app/views/users/friend_responses.html.erb

```diff
@@ -0,0 +1,19 @@
+<table>
+  <thead>
+  <tr>
+    <th>Name</th>
+    <th>Status</th>
+    <th>Actions</th>
+  </tr>
+  </thead>
+  <% @responses.each do |response| %>
+    <tr>
+      <th><%= response.name %></th>
+      <th><%= response.status %></th>
+      <% if response.status == 'Pending' %>
+        <th><%= link_to 'Accept', accept_friend_request_path(response), method: :post %></th>
+        <th><%= link_to 'Reject', reject_friend_request_path(response), method: :post %></th>
+      <% end %>
+    </tr>
+  <% end %>
```

Comment:
- [ ] There should be absolutely NO logic in views. Views are only for displaying information that became available to it from its corresponding controller not generating it.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright)
I have gone ahead and made some comments on your codebase see affected files below:

**Don't know where to start or how to?** See  [Rails View Helpers Best Practices](https://medium.com/@acushla4real/rails-view-helpers-best-practices-guide-d7d7b6db3db4)